Formula/libmonome.rb

```diff
@@ -16,15 +19,24 @@ class Libmonome < Formula
   end
 
   depends_on "liblo"
+  depends_on "python@3.10"
```

Comment:
Does it require python in runtime, or we can add it only for the build?

Reply:
Good point. We're not installing the Python bindings and it doesn't require Python during runtime, so Python should just be a build dependency. I also edited the description to better reflect what the formula does/contains. On a related note, they're considering removing the Python bindings: monome/libmonome#66.